### ISSUE TITLE
Error on Android N

### ISSUE THREAD

**Issue by vogi** [paypal/sdk-packages#105]
I use chain payment with the PayPal MPL on the Sandbox. On devices with Lollipop it works, on my Nexus 5x with Nougat (7.1.1) i get "An unexpected error has occured". Same user, same APK. Debugger output is "Login: Error during call to log in. MODE_WORLD_WRITEABLE no longer supported  -> unexpected error". The library with version 1.5.6.58 seems to be incompatible with new Android Version?

**Comment by ganlaw** [comment]
Did you manage to solve this issue? I am having a very similar issue on my N device?

**Comment by xiaoleih41** [comment]
We are looking into this issue. Thanks,

**Comment by gabrieldelmas** [comment]
Any update ?

**Comment by jdsepulveda** [comment]
I'm having the same problems with Nougat (7.1.1) Is this issue solved?

**Comment by gabrieldelmas** [comment]
As far as I know, this issue is not solved. I had no feedback from PayPal on this issue. Finally, I had to move on to something else for our payment system.

I investigated, here are some informations maybe that could help the PayPal team. The source code is obfuscated so I don’t know which class is concerned  but during a XML parsing the library calls the method Context#openFileOutput with a deprecated constant MODE_WORLD_WRITEABLE and throws an exception. So maybe if the library use another constant such as MODE_PRIVATE it could fix this issue.
